Selenium.Axe/Selenium.Axe/AxeRunOptions.cs

```diff
@@ -54,5 +70,43 @@ public class AxeRunOptions
         [JsonProperty("rules")]
         public Dictionary<string, RuleOptions> Rules { get; set; }
 
+        /// <summary>
+        /// Limit which result types are processed and aggregated. An approach you can take to reducing the time is use the resultTypes option. 
+        /// For eg, when set to [ResultTypes.Violations], scan results will only have the full details of the violations array and 
+        /// will only have one instance of each of the inapplicable, incomplete and pass arrays for each rule that has at least one of those entries. 
+        /// This will reduce the amount of computation that axe-core does for the unique selectors.
+        /// </summary>
+        [JsonProperty("resultTypes")]
+        public HashSet<ResultType> ResultTypes { get; set; }
+
+        /// <summary>
+        /// Returns xpath selectors for elements
+        /// </summary>
+        [JsonProperty("xpath")]
+        public bool? XPath { get; set; }
```

Comment:
I don't think this is actually functional without also adding an equivalent property to `AxeResultNode`. Consider updating the IntegrationTests to catch this?

Reply:
good catch.